README.md

```diff
@@ -883,6 +903,23 @@ __AWESOME__ apps counter: __130__ 🎉
 - [x] [GitHub](https://github.com/gauravjot/android-noad-music-player)
 - [ ] Official page
 
+## Music Scrobbler
```

Comment:
I read about what `Scrobbler` means, but still I can't understand the practical standing point. Could you please explain this to me?

Reply:
People have spotify and get their music report ib the end of the year but all people are not caged by spotify, some people host their own musis server and scrobblers their music to last.fm or libre.fm. They get music report every day. 
In simple word it is a simple way of sharing one's music listening activity with the world.